### ISSUE TITLE
Please include the XML document used in your example

### ISSUE THREAD

**Issue by donslattery** [dotnet/docs#22249]
It would be really helpful to have the XML document you used in your example was visible on the page so we could have some context as to where you're picking up the elements from.  I'm referring specifically to this part of the code:
from item in purchaseOrder.Descendants("Item")
                               select (string) item.Attribute("PartNumber");

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d01ee7ce-9675-7bbc-430d-5a28876a6d06
* Version Independent ID: f718ba40-50aa-0b5f-4ff2-73b3430b37f4
* Content: [Overview - LINQ to XML](https://docs.microsoft.com/en-us/dotnet/standard/linq/linq-xml-overview)
* Content Source: [docs/standard/linq/linq-xml-overview.md](https://github.com/dotnet/docs/blob/master/docs/standard/linq/linq-xml-overview.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **dotnetcontent**

---
[Associated WorkItem - 564183](https://dev.azure.com/msft-skilling/Content/_workitems/edit/564183)

**Comment by gewarren** [comment]
@donslattery The XML doc is linked in this sentence.

"For example, you might have a typical XML purchase order as described in Sample XML file: Typical purchase order."

**Comment by donslattery** [comment]
Thanks for pointing that out.  So what you're saying is that including that content on this page would make absolutely no improvement or clarify it at all for casual readers led here by search engines and that the link buried in the middle of multiple lines of text is sufficient to point folks to a critical piece of information about the example?  Nor would it fit into the established examples where content like this IS included on other dot net pages.

**Comment by gewarren** [comment]
No, I didn’t intend for you to make those assumptions just by pointing out that the XML was linked. It was unclear whether you’d found the XML or not. We can add this to our backlog or you can submit a pull request that puts the XML inline if you feel it would improve the article.

![image](https://user-images.githubusercontent.com/24882762/103949539-454d5080-50f0-11eb-8734-99a50ee74b78.png)

**Comment by donslattery** [comment]
Understood.  Thank you for clarifying.  When I read the article, I overlooked the link to the XML because the narrative is usually less helpful than the code.  My concern is that more people do this than search through the text when coming from a search from a search engine.  In this case, I think including the XML improves the article.  As far as updates, I think it would be better if it was added to the backlog.  My lack of familiarity with pull requests is more likely to mess something up than to improve it.
